### PR TITLE
Pruning table cache

### PR DESCRIPTION
**Pull request by V-Wong**

### CODE REVIEW

File: src/thistlethwaite.rs [hunk 1]
```diff
@@ -19,105 +21,139 @@ pub fn solve(cube: &impl Cube) -> Option<Vec<Move>> {
...
+                S(3, U, 1), S(3, U, 3), S(3, U, 7), S(3, U, 9),
+                S(3, D, 1), S(3, D, 3), S(3, D, 7), S(3, D, 9)
+            ];
+        
+            let eo_ud_pieces = [
+                S(3, U, 2), S(3, U, 4), S(3, U, 6), S(3, U, 8),
+                S(3, D, 2), S(3, D, 4), S(3, D, 6), S(3, D, 8)
+            ];
+        
+            let eo_e_piecs = [
```
Comment (stanleydesu) [inline src/thistlethwaite.rs:69]: `eo_e_pieces` instead?

File: src/thistlethwaite.rs [hunk 2]
```diff
@@ -19,105 +21,139 @@ pub fn solve(cube: &impl Cube) -> Option<Vec<Move>> {
...
+                .concat();
+
+            let face = |f| if f == B { F }
+                           else if f == L { R }
+                           else { f };
+
+            Box::new(move |i: i32, _| 
+                    if cp_pieces.contains(&i) { [U, R, F, D, L, B][(i / 9) as usize] }
+                    else if ep_pieces.contains(&i) { face([U, R, F, D, L, B][(i / 9) as usize]) }
+                    else { X }
```
Comment (stanleydesu) [inline src/thistlethwaite.rs:126]: Indentation different to line 74. Line 74 is preferred?

**Review by stanleydesu — CHANGES_REQUESTED** [review]
Other than a typo and indentation fix, it looks good.

File: src/thistlethwaite.rs (hunk 1 above)
Comment (V-Wong) [reply src/thistlethwaite.rs]: Fixed

File: src/thistlethwaite.rs (hunk 2 above)
Comment (V-Wong) [reply src/thistlethwaite.rs]: Fixed.

**Pull request merged**